src/test/scala/cromwell/HelloWorldActorSpec.scala

```diff
@@ -47,25 +47,24 @@ class HelloWorldActorSpec extends CromwellSpec(ActorSystem("HelloWorldActorSpec"
   val TestExecutionTimeout = 5000 milliseconds
 
   "A WorkflowActor" should {
-
     "start" in {
+      val binding = WdlBinding.process(HelloWorld.WdlSource)
+      val coercedInputs = binding.coerceRawInputs(HelloWorld.RawInputs).get
+      /*
+        The TestFSMRef is kind of quirky, defining it here instead of the buildWorkflowActor function. It could
+        be generalized a bit but it is probably not worth the hassle for a test class
+       */
+      val fsm = TestFSMRef(new WorkflowActor(UUID.randomUUID(), binding, coercedInputs, new LocalBackend))
+      assert(fsm.stateName == WorkflowSubmitted)
+      fsm ! Start
       within(TestExecutionTimeout) {
-        val workflowActor = buildWorkflowActor("started")
-        startingCallsFilter("hello").intercept {
-          workflowActor ! Start
-          expectMsgPF() {
-            case Started => ()
-          }
-          expectMsgPF() {
-            case Failed(t) =>
-              fail(t)
-            case Done(outputs) =>
-              val outputName = "hello.hello.salutation"
-              val salutation = outputs.getOrElse(outputName, throw new RuntimeException(s"Output '$outputName' not found."))
-              val actualOutput = salutation.asInstanceOf[WdlString].value.trim
-              actualOutput shouldEqual "Hello world!"
-          }
-        }
+        awaitCond(fsm.stateName == WorkflowRunning)
+        awaitCond(fsm.stateName == WorkflowSucceeded)
```

Comment:
Although we probably shouldn't choose a paradigm based on how much prettier it makes the test code, this does make the test code a *lot* prettier.  :smile: